index.md

```diff
@@ -103,6 +103,7 @@ color by default via `NO_COLOR`.
 | [crayon](https://github.com/r-lib/crayon) | R package for colored terminal output | [2018-02-08 / 1.5.1](https://github.com/r-lib/crayon/commit/700800135d04408bf1c99426b3fec9a4073b8a97) |
 | [crayon.js](https://github.com/crayon-js/crayon) | Modular Deno module and NPM package for terminal styling | [2020-04-10 / 1.0.0](https://github.com/crayon-js/crayon/commit/c7f8b1c4a1acecd93592bd330c04da70c0dc7051) |
 | [crossterm](https://github.com/crossterm-rs/crossterm) | Cross-platform terminal manipulation library in Rust | [2023-08-06 / 0.27](https://github.com/crossterm-rs/crossterm/releases/tag/0.27.0) |
+| [crystal-lang colorize module](https://crystal-lang.org/) | Crystal module to change the fore- and background colors and text decorations when rendering text on terminals. | [2022-06-06 / 1.5.0](https://github.com/crystal-lang/crystal/releases/tag/1.5.0) |
```

Comment:
Please shorten the description and remove the trailing period

Reply:
Done.